src/player.h

```diff
@@ -1255,6 +1258,7 @@ class Player final : public Creature, public Cylinder
 	uint16_t lastStatsTrainingTime = 0;
 	uint16_t staminaMinutes = 2520;
 	uint16_t maxWriteLen = 0;
+	uint16_t baseExpGain = 100;
```

Comment:
This is misleading as this does not affect exp gain rate at all and is only used for display purposes

Reply:
This is in no way related to players baseExpGain, if you set it to 200 player won't start getting more XP. All it does is update value in Client (display).

Calling player:setBaseExpGain does not affect the exp gain.

Reply:
any name suggestions?